stix_shifter_modules/azure_log_analytics/configuration/lang_en.json

```diff
@@ -1,21 +1,21 @@
 {
   "connection": {
     "host": {
-      "label": "Management IP address or Hostname",
-      "description": "Specify the IP address or hostname of the data source"
+      "label": "Management IP address or hostname",
+      "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
```

Comment:
you need to remove the mention of `IBM Cloud Pak for Security` from the description. Maybe just revert back the change. This applies to all `host` description changes in this pr

Reply:
I changed all of the host to use 
`Specify the IP address or hostname of the data source`